### PR TITLE
use build for koji id if already exists, fix use json method instead of attribute from get_build

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by mlangsdorf** [comment]
summary is insufficient. Fix what?

I also don't follow how passing osbs to BuildResponse in a few more instances affects using using the build id for the koji id. What am I missing?

File: tests/test_api.py [hunk 1]
```diff
@@ -2017,15 +2020,15 @@ def test_use_already_created_build(self, openshift, variation, delegate_method,
...
+                .and_return(MockResponse))
 
         if exc:
             with pytest.raises(OsbsException) as exc_info:
                 osbs_obj.create_build(**kwargs)
             assert "Multiple builds 2 for koji task id %s" % koji_task_id in exc_info.value.message
         else:
             build_response = osbs_obj.create_build(**kwargs)
-            assert build_response.json() == {'spam': 'maps'}
+            assert build_response.json == {'spam': 'maps'}
```
[the remark this answers is not in the document]
Comment (lcarva) [reply tests/test_api.py]: ok, you're right. BuildResponse object doesn't have `json()` method, it has a `json` attribute.
You could avoid the mock here and just create a BuildResponse object directly :)

[the remark this answers is not in the document]
Comment (rcerven) [reply tests/test_api.py]: right updated

**Comment by rcerven** [comment]
not updating release note, as this is fix for unreleased change

**Pull request merged**